openid-connect-common/pom.xml

```diff
@@ -62,6 +62,12 @@
       		<groupId>org.springframework.security.oauth</groupId>
  		 	<version>2.0.0.M2</version>
       		<artifactId>spring-security-oauth2</artifactId>
+            <exclusions>
+               <exclusion>
+                  <groupId>org.springframework.security</groupId>
+                  <artifactId>spring-security-config</artifactId>
+               </exclusion>
+            </exclusions>
```

Comment:
It's not related to the bug fix but it prevents 2 versions of spring-security-config appearing in my WEB-INF/lib.